src/components/cart.js

```diff
@@ -190,7 +190,8 @@ export default class Cart extends Component {
   }
 
   onCheckout() {
-    this.checkout.open(this.model.checkoutUrl);
+	  this._userEvent('openCheckout');
```

Comment:
For consistency, could you indent with 2 spaces instead of a tab?

Reply:
hi @spencercanner, sorry for taking so long to get back to you. I saw @tombennet has a new PR open to address the coding style issue and I'm happy to close this PR. Please let me know if you need any help. Thanks.